pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -280,9 +293,87 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (resul
 		return reconcile.Result{}, err
 	}
 
+	rolResult, err := r.reconcileOwnerReferences(cd, cdLog)
+	if err != nil {
+		cdLog.WithError(err).Error("Failed to reconcile objects owned by this cluster deployment.")
+		return reconcile.Result{}, err
+	}
+
+	if rolResult != nil {
+		cdLog.WithError(err).Error("Reconciled objects owned by this cluster deployment.")
```

Comment:
This should not be an error-level message. Or, at a minimum, it should not use `WithError` since `err` will always be nil.